cmd/minikube/cmd/start_flags.go

```diff
@@ -118,6 +118,8 @@ const (
 	defaultSSHUser          = "root"
 	defaultSSHPort          = 22
 	listenAddress           = "listen-address"
+	extraDisks              = "extra-disks"
+	extraDiskSize           = "extra-disk-size"
```

Comment:
for the sake of not having too many flags, what do u think/feel about re-suing the existing the flag we have `disk-size`
```
$ minikube start --help | grep disk
      --disk-size='20000mb': Disk size allocated to the minikube VM (format: <number>[<unit>], where unit = b, k, m or g).
```

so if user passes this flag we will use disk-size value as the extra disk size and if they pass --disk-size it will be used for both main disk and extra disk, is that something acceptable ?

Reply:
I like that idea, and I thought about doing that. It is more flexible to have separate options, but it does add complexity for the user. I keep waffling on this. For all of the use-cases I have in mind, I don't think there is a very strong reason to have the extra parameter for development just to save a few 10s of gigabytes on a hard disk. I think I will take your suggestion to simply re-use the `disk-size` parameter for extra disks as well, and if that ends up being insufficient, it can always be changed in the future.